repose-aggregator/components/filters/herp-filter/src/main/scala/org/openrepose/filters/herp/HerpFilter.scala

```diff
@@ -51,6 +52,7 @@ import scala.util.matching.Regex
 
 @Named
 class HerpFilter @Inject()(configurationService: ConfigurationService,
+                           openTracingService: OpenTracingService,
```

Comment:
Why? You've added a bunch of deps to this filter, but don't seem to use them at all?